src/view/container.tsx

```diff
@@ -25,7 +26,11 @@ export function Container() {
     // is available in the state
     dispatch(actions.SetHeader(config.header));
 
-    processPipeline();
+    if ((config.search as SearchConfig)?.keyword) {
+      //skipping initial pipeline as search keyword is present and search plugin will handle the initial data laod
+    } else {
+      processPipeline();
```

Comment:
Why do we need to skip this?

Reply:
If you look into the referenced issue #1330 that has more details to it. I see that you pushed fixes related to that already. I didn't get a chance to validate the new build, but if #1330 is already fixed in your other fixes then we don't need this PR.